examples/04-sage-planets/index.ts

```diff
@@ -0,0 +1,177 @@
+import { Program, BN } from '@project-serum/anchor';
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import bs58 from 'bs58';
+
+import { byteArrayToString, readAllFromRPC, readFromRPC } from "@staratlas/data-source";
+import { MineItem, Planet, Resource, SAGE_IDL } from "@staratlas/sage";
+
+import { newConnection, newAnchorProvider } from '../../shared/anchor-setup';
+import { generateKeypair } from '../../shared/wallet-setup';
+
+import { Table } from 'console-table-printer';
+
+const SAGE_PROGRAM_ID = 'SAGE2HAwep459SNq61LHvjxPk4pLPEJLoMETef7f7EE';
+
+const mainFunction = async (connection: Connection, myWallet: Keypair) => {
```

Comment:
just `main` would rather be convention

Reply:
See commit [7100121](https://github.com/staratlasmeta/star-atlas-cookbook/pull/1/commits/710012175c04d238db757ca99467af89103325c0)

Reply:
> again just copied what was done in first example script but will change to `main`.

Yea I am aware. I promised @cuniculture a review of this initial code, so your PR was just perfectly fine to add my two cents. All minor comments anyways. 

Thanks for contribution.